airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/LowCreditBalanceHint/LowCreditBalanceHint.tsx

```diff
@@ -0,0 +1,35 @@
+import { faCreditCard, faWarning } from "@fortawesome/free-solid-svg-icons";
+import { FormattedMessage } from "react-intl";
+
+import { InfoBox } from "components/ui/InfoBox";
+
+import { useGetCloudWorkspace } from "packages/cloud/services/workspaces/CloudWorkspacesService";
+import { useCurrentWorkspace } from "services/workspaces/WorkspacesService";
+
+import styles from "./LowCreditBalanceHint.module.scss";
+
+export const LOW_BALANCE_CREDIT_TRESHOLD = 20;
+
+export const LowCreditBalanceHint: React.FC<React.PropsWithChildren<unknown>> = ({ children }) => {
+  const workspace = useCurrentWorkspace();
+  const cloudWorkspace = useGetCloudWorkspace(workspace.workspaceId);
+
+  if (cloudWorkspace.remainingCredits > LOW_BALANCE_CREDIT_TRESHOLD) {
+    return null;
+  }
+
+  const status = cloudWorkspace.remainingCredits === 0 ? "zeroBalance" : "lowBalance";
```

Comment:
I think in some cases we have workspaces with negative `remainingCredits` (see my cloud workspace [here](https://cloud.airbyte.io/workspaces/c508c8ed-38d2-4645-b8c6-dd9140316279) for an example), so I think we should do a `<= 0` comparison here.